deploy.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-fldist deploy_app --env krasnodar -s Fs6nQaGEsM5EgnprUbUtoLYWhUC8o6QK1gseP9pfhzUm --node-id 12D3KooWSD5PToNiLQwKDXsu8JSysCwUt8BVUJEqCHcDe7P5h45e -i app/app.config.json -o client/src/app.json
\ No newline at end of file
+fldist deploy_app --env krasnodar --node-id 12D3KooWSD5PToNiLQwKDXsu8JSysCwUt8BVUJEqCHcDe7P5h45e -i app/app.config.json -o client/src/app.json
```

Comment:
Let's avoid using `12D3KooWSD5PToNiLQwKDXsu8JSysCwUt8BVUJEqCHcDe7P5h45e`, it's already overloaded. So is `12D3KooWR4cv1a8tv7pps4HH6wePNaK6gf1Hww5wcCMzeWxyNw51`.

Better use `12D3KooWKnEqMfYo9zvfHmqTLpLdiHXPe4SVqUWcWHDJdFGrSmcA` and others.